package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "dropbox-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "tsc && node ./dist/index.js",
+    "serve": "./dist/index.js",
+    "build": "tsc"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/axios": "^0.14.0",
+    "@types/elastic-apm-node": "^2.7.0",
```

Comment:
`elastic-apm-node` provides its own type definitions, so you don't need @types/elastic-apm-node installed. 😄

Reply:
:+1: